package.json

```diff
@@ -53,6 +56,7 @@
   },
   "devDependencies": {
     "@docusaurus/eslint-plugin": "^2.1.0",
+    "@redocly/cli": "^1.0.0-beta.124",
```

Comment:
Is it a bad idea to add `redocly` here as a dependency? Would it just be better to use the `npx` method of running the bundle command?

Reply:
I think it's ok to include as dependency

Reply:
Sounds good to me. Thanks!